docs/_templates/layout.html

```diff
@@ -0,0 +1,50 @@
+
+{% extends "!layout.html" %}
+{%- block content %}
+{%- if theme_fixed_sidebar|lower == 'true' %}
+  <div class="document">
+    {{ sidebar() }}
+    {%- block document %}
+      <div class="documentwrapper">
+      {%- if render_sidebar %}
+        <div class="bodywrapper">
+      {%- endif %}
+
+          {%- block relbar_top %}
+            {%- if theme_show_relbar_top|tobool %}
+              <div class="related top">
+                &nbsp;
+                {{- rellink_markup () }}
+              </div>
+            {%- endif %}
+          {% endblock %}
+
+          <div class="body" role="main">
+          	<div class="admonition" id="python2-eol">
+          	 As of January 1, 2020 this library no longer supports Python 2 on the latest released version.
+          	 Library versions released prior to that date will continue to be available. For more information please
+          	 visit <a href="https://cloud.google.com/python/docs/python2-sunset/">Python 2 support on Google Cloud</a>.
+          	</div>
+            {% block body %} {% endblock %}
+          </div>
```

Comment:
Since we won't have any supported releases before 1/1/20 do we need this boilerplate? I know the other googleapis repos have it, but maybe we can lose it here.

And maybe we can come up with a better way for these repos to share a doc style than copy-pasting the template to each new repo, but that's a problem for another PR.

Reply:
Makes sense. The `admonition` div is removed, but I'd leave the `body` for now.